ai-platform-unified/notebooks/unofficial/hyperparameter_tuning/distributed-hyperparameter-tuning.ipynb

```diff
@@ -0,0 +1,879 @@
+{
```

Comment:
<p>you only cleaned up the bucket, not the hyperparam tuning job</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
No model is created, just a training pipeline. Does that need to be deleted as well? I don't think training pipelines result in extra costs if they are not deleted. If it does need to be deleted, do you know how to do that with gsutil?